onnxruntime/core/providers/tensorrt/tensorrt_execution_provider.h

```diff
@@ -74,16 +71,23 @@ class TensorrtExecutionProvider : public IExecutionProvider {
   std::shared_ptr<KernelRegistry> GetKernelRegistry() const override;
 
   void SetMaxBatchSize(const int batch_size) {
-      max_batch_size_ = batch_size;
+    max_batch_size_ = batch_size;
   }
 
   void SetMaxWorkspaceSize(const size_t workspace_size) {
-      max_workspace_size_ = workspace_size;
+    max_workspace_size_ = workspace_size;
   }
-    
+
+  std::unique_ptr<IndexedSubGraph> GetSubGraph(SubGraph_t graph_nodes_index, int& kernels_index,
+                                               const onnxruntime::GraphViewer& graph) const;
+
+  SubGraphCollection_t GetSupportedList(SubGraphCollection_t supported_nodes_list, int iterations, const int& max_iterations,
+                                        bool& early_termination, const onnxruntime::GraphViewer& graph) const;
```

Comment:
early_termination seems to be an output parameter?
if so, it should follow the inputs in the method signature.
also it shouldn't be reference, should be pointer for outputs.